### DISCUSSION TITLE
Configurar quantidade de parcelas para integração do tipo Cardform

### DISCUSSION THREAD

**Discussion by asd123ea** [mercadopago/sdk-js#200] [Q&A] [answered]
Olá, gostaria de saber se existe a possibilidade de configurar a quantidade de parcelas (via codigo, no BackOffice da MPago, etc...) que é exibida na compra por cartão de crédito utilizando a integração Cardform do Checkout Transparente.

Caso não existe, eu teria que mudar o tipo de integração que utilizo? Por exemplo Métodos Core do Checkou Transparente eu vi na documentação que suporta essa configuração via código.

Caso possam providenciar a documentação evidenciando os fatos, seria ótimo. Não achei na documentação do Cardform a afirmação que não suporta.

**Comment by AleFossati** [comment, marked as answer]
Olá @asd123ea, para o seu caso de uso os Bricks parecem ser o produto adequado.
Se quiser disponibilizar apenas pagamentos com cartão, pode usar o [Card Payment Brick](https://www.mercadopago.com.br/developers/pt/docs/checkout-bricks/card-payment-brick/introduction) e [configurar o parcelamento](https://www.mercadopago.com.br/developers/pt/docs/checkout-bricks/card-payment-brick/advanced-features/configure-installments) da forma indicada no link.
Para oferecer múltiplos meios de pagamento, pode usar o [Payment Brick](https://www.mercadopago.com.br/developers/pt/docs/checkout-bricks/payment-brick/introduction), onde também é possível [configurar o parcelamento](https://www.mercadopago.com.br/developers/pt/docs/checkout-bricks/payment-brick/advanced-features/configure-installments) da forma indicada no link.
Infelizmente o CardForm não suporta configurações de parcelamento.
